src/graphql/resolvers/login-resolver.ts

```diff
@@ -0,0 +1,30 @@
+import { verify } from '../../services/login/verification-service';
+
+/**
+ *Resolver obtains the unsigned_address from the query request
+ *then it calls the loginDataInput to obtained the singed_address
+ *which then is returned as query.
+ */
+
+const resolver = {
+	Mutation: {
+		verify: (_: any, args: { signed_message: string }, context: any) => {
+			console.log(args.signed_message);
+			const ret_address = verify(args.signed_message);

```

Comment:
Can we check whether this retAddress is nonempty/nonnull and return appropriately.